source/lexbor/ports/windows_nt/lexbor/fs.c

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (C) 2018 Alexander Borisov
+ *
+ * Author: Alexander Borisov <lex.borisov@gmail.com>
+ *         Vincent Torri <vincent.torri@gmail.com>
```

Comment:
Only `Vincent Torri` (I did not develop this file :)):
* Author: Vincent Torri <vincent.torri@gmail.com>